src/Features/Core/Portable/AddPackage/InstallPackageDirectlyCodeActionOperation.cs

```diff
@@ -72,7 +72,8 @@ internal class InstallPackageDirectlyCodeActionOperation : CodeActionOperation
         {
             return _installerService.TryInstallPackage(
                 workspace, _document.Id, _source, _packageName,
-                _versionOpt, _includePrerelease, cancellationToken);
+                _versionOpt, _includePrerelease,
+                progressTracker, cancellationToken);
```

Comment:
once https://github.com/dotnet/roslyn/pull/55682 goes in, this can be async end to end.